integration/shareddata/composites.go

```diff
@@ -382,3 +382,28 @@ var ArrayDocuments = &Values[string]{
 		},
 	},
 }
+
+// TopLevelFieldsIntegers contains documents with several top level fields with integer values.
+var TopLevelFieldsIntegers = &TopLevelValues[string]{
+	name:     "TopLevelFieldsIntegers",
+	backends: []string{"ferretdb-pg", "ferretdb-tigris", "mongodb"},
+	validators: map[string]map[string]any{
+		"ferretdb-tigris": {
+			"$tigrisSchemaString": `{
+				"title": "%%collection%%",
+				"primary_key": ["_id"],
+				"properties": {
+					"foo": {"type": "integer", "format": "int32"},
+					"bar": {"type": "integer", "format": "int32"},
+					"_id": {"type": "string"}
+				}
+			}`,
+		},
+	},
+	data: map[string]fields{
+		"int32-two": {
+			"foo": int32(1),
+			"bar": int32(2),
+		},
+	},
+}
```

Comment:
I'm seeing quite a lot of test failures due to the order of fields `foo` and `bar`. I think it's expected from this because fields are design to be random order by using map. There can be multiple way of handling this, either ensure fields are the specific order, tests sorts response before compare or something else.

```
2023-04-17T18:40:02.3596105Z             --- FAIL: TestUpdateFieldCompatSetOnInsert/DocArrFieldNotExist/TestUpdateFieldCompatSetOnInsert-DocArrFieldNotExist_TopLevelFieldsIntegers/int32-two (0.04s)
2023-04-17T18:40:02.3596429Z                 update_field_compat_test.go:968: 
2023-04-17T18:40:02.3597076Z                            Error Trace:    /home/runner/work/FerretDB/FerretDB/internal/util/testutil/equal.go:42
2023-04-17T18:40:02.3597968Z                                                                    /home/runner/work/FerretDB/FerretDB/integration/helpers.go:143
2023-04-17T18:40:02.3598942Z                                                                    /home/runner/work/FerretDB/FerretDB/integration/update_compat_test.go:143
2023-04-17T18:40:02.3599260Z                            Error:          Not equal: 
2023-04-17T18:40:02.3599574Z                                            expected: {
2023-04-17T18:40:02.3599870Z                                              "$k": [
2023-04-17T18:40:02.3600186Z                                                "_id",
2023-04-17T18:40:02.3600503Z                                                "foo",
2023-04-17T18:40:02.3600806Z                                                "bar"
2023-04-17T18:40:02.3601064Z                                              ],
2023-04-17T18:40:02.3601523Z                                              "_id": "int32-two",
2023-04-17T18:40:02.3601844Z                                              "foo": 1,
2023-04-17T18:40:02.3602201Z                                              "bar": 2
2023-04-17T18:40:02.3602418Z                                            }
2023-04-17T18:40:02.3602725Z                                            actual  : {
2023-04-17T18:40:02.3603023Z                                              "$k": [
2023-04-17T18:40:02.3603338Z                                                "_id",
2023-04-17T18:40:02.3603647Z                                                "bar",
2023-04-17T18:40:02.3603947Z                                                "foo"
2023-04-17T18:40:02.3604201Z                                              ],
2023-04-17T18:40:02.3604658Z                                              "_id": "int32-two",
2023-04-17T18:40:02.3605022Z                                              "bar": 2,
2023-04-17T18:40:02.3605392Z                                              "foo": 1
2023-04-17T18:40:02.3605625Z                                            }
2023-04-17T18:40:02.3605996Z                                            --- expected
2023-04-17T18:40:02.3606289Z                                            +++ actual
2023-04-17T18:40:02.3606667Z                                            @@ -3,8 +3,8 @@
2023-04-17T18:40:02.3606991Z                                                 "_id",
2023-04-17T18:40:02.3607344Z                                            -    "foo",
2023-04-17T18:40:02.3607687Z                                            -    "bar"
2023-04-17T18:40:02.3607993Z                                            +    "bar",
2023-04-17T18:40:02.3608294Z                                            +    "foo"
2023-04-17T18:40:02.3608557Z                                               ],
2023-04-17T18:40:02.3609022Z                                               "_id": "int32-two",
2023-04-17T18:40:02.3609396Z                                            -  "foo": 1,
2023-04-17T18:40:02.3609751Z                                            -  "bar": 2
2023-04-17T18:40:02.3610063Z                                            +  "bar": 2,
2023-04-17T18:40:02.3610374Z                                            +  "foo": 1
2023-04-17T18:40:02.3610614Z                                             }
```

Reply:
Fixed it.